src/RRule.php

```diff
@@ -2016,11 +2016,21 @@ static protected function i18nList(array $array, $and = 'and')
 	 * Will load the basic first (e.g. "en") and then the region-specific if any
 	 * (e.g. "en_GB"), merging as necessary.
 	 * So region-specific translation files don't need to redefine every strings.
+	 *
+	 * @param string $locale
+	 * @param string $fallbackLocale
+	 *
+	 * @return array
+	 * @throws \InvalidArgumentException
 	 */
-	static protected function i18nLoad($locale)
+	static protected function i18nLoad($locale, $fallbackLocale = 'en_US')
 	{
-		if ( ! preg_match('/^([a-z]{2})(_[A-Z]{2})?$/', $locale, $matches) ) {
-			throw new \InvalidArgumentException('The locale option does not look like a valid locale: '.$locale);
+		if ( ! preg_match('/^([a-z]{2})(_[A-Z]{2})?(_[A-Z]*)?$/', $locale, $matches) ) {
+			if ( $fallbackLocale ) {
+				$locale = $fallbackLocale;
```

Comment:
I'm not sure this test should be here. You should use the fallback only when the locale *file* doesn't exist. However, if you pass a wrong locale to the function (such as "foobar"), it should be strict and throw an `invalidArgumentException`.